compiler-rt/lib/scudo/standalone/secondary.h

```diff
@@ -321,8 +321,6 @@ class MapAllocatorCache {
         }
         CachedBlock PrevEntry = Quarantine[QuarantinePos];
         Quarantine[QuarantinePos] = Entry;
-        if (OldestTime == 0)
```

Comment:
It looks like you are combining the removal of OldestTime with this cl. Did you mean to do that?

Reply:
That's an unused variable since we adopt LRU and I said it's fine to include the removal in this CL

Reply:
That sounds fine to me. But you should mention it in the description.

By the way, there is a misspelled word (tthe) in the description too.

Reply:
It could technically be used as a heuristic for when to release in the quarantine, but I think there was also a bug report about how this value is not always accurate. I'll add a comment about the removal in the commit message.